proto/executor.proto

```diff
@@ -375,6 +381,10 @@ message TiFlashScanContext {
 	optional uint64 total_vector_idx_read_others_time_ms = 108;
 }
 
+message TiFlashPipelineWaitSummary {
```

Comment:
maybe rename this to TiFlashWaitSummary and put all the tiflash wait info into it?

Reply:
OK, addressed